apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts

```diff
@@ -47,17 +47,121 @@ const validators = [
   },
 ];
 
-for (const account of e2eDelegationAccounts) {
-  test.describe("Delegate", () => {
+test.describe("Delegate flows", () => {
+  test.beforeAll(async () => {
+    if (new Date().getDay() === 1) {
```

Comment:
Shouldn't we also add a condition to broadcast only scheduled run or a condition to do on demand for release ?
(Same for all below)

Reply:
indeed, added an additional check
will test tomorrow